app/uk/gov/hmrc/timetopayproxy/models/CreatePlanRequest.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.timetopayproxy.models
+
+
+import play.api.libs.json.Json
+
+import java.time.LocalDate
+
+final case class PlanInstalment(dutyId: DutyId,
+                                debtId: DebtId,
+                                dueDate: LocalDate,
```

Comment:
why do need to repeat dutyId and debtIt for each instalment?

Reply:
good point, i'll raise with Jorma. Maybe debtId is enough